src/Panel/Plugins.php

```diff
@@ -42,6 +44,8 @@ public function files(): array
 		foreach (App::instance()->plugins() as $plugin) {
 			$this->files[] = $plugin->root() . '/index.css';
 			$this->files[] = $plugin->root() . '/index.js';
+			// used by kirbyup during plugin development
+			$this->files[] = $plugin->root() . '/_index.mjs';
```

Comment:
I think I'd prefer it without the underscore. If we want to make it clear it is only for development, we should maybe use something like `index.dev.mjs`.

Reply:
Done :)